testdata/p4tc_samples_outputs/add_entry_3_example_control_blocks.c

```diff
@@ -26,8 +26,8 @@ struct __attribute__((__packed__)) MainControlImpl_ipv4_tbl_1_value {
         struct {
         } MainControlImpl_next_hop;
         struct __attribute__((__packed__)) {
-            u64 dmac;
-            u64 smac;
+            u8 dmac[6];
```

Comment:
I understand doing the same thing we want for keys to action parameters, it's consistent
However this will break the existing kernel code, because it expects action parameters to be as they were before
So, in this case, `dmac` and `smac` should be `u64`
In the future we might apply the same change for action parameters, but as of now action parameters should remain the same

Reply:
As agreed, we will keep the alignment same for action parameters as well to maintain consistency.